src/tools/Reporting/FailureReporting/FailureReporting.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I believe that you will need to target netcoreapp3.1 here, because by default when you use net standard you get a dll, and not a standalone executable. cc @billwert

Reply:
Correct. This is a library project, not a console exe. You need to add `<OutputType>Exe</OutputType>` as well, and for completeness so it doesn't cause problems in the future I would use our standard version setting technique, seen here:

https://github.com/dotnet/performance/blob/master/src/tools/ScenarioMeasurement/SizeOnDisk/SizeOnDisk.csproj#L5-L7

Reply:
Do I just change those lines, or is there more I need to do?